src/aiNpc/Stats/genesis.data.ts

```diff
@@ -0,0 +1,17 @@
+//Forbiden imports here
+
+
+//Global access to the game data
+class GenesisData {
+
+    readonly startPlayTime: number = Date.now()
+
+    private static instanceRef: GenesisData;
+
+    bdeploy: boolean = false;
```

Comment:
@AnisLahouar  spelling mistake?

Reply:
in all honesty, idk, i just used the same file from onboarding and removed the extra elements, that one slipped by me